server/bin/pbench-dispatch.sh

```diff
@@ -338,22 +333,37 @@ while read tbmd5; do
         if [[ ${toterr} -gt 0 ]]; then
             # We have had some errors while processing this tar ball, so count
             # this as a partial success.
-            log_info "${TS}: ${controller}/${resultname}: success (partial)" "${status}"
+            log_info "${TS}: ${controller}/${resultname}: success (partial)"
             (( npartialsucc++ ))
         else
-            log_info "${TS}: ${controller}/${resultname}: success" "${status}"
+            log_info "${TS}: ${controller}/${resultname}: success"
         fi
     fi
 done < ${list}
 
-summary_text="Processed ${ntotal} result tar balls, ${ntbs} successfully"\
-" (${npartialsucc} partial), with ${nquarantined} quarantined tar balls,"\
-" ${ndups} duplicately-named tar balls, and ${nerrs} errors."
-
-log_info "${PROG}.${TS}(${PBENCH_ENV}): ${summary_text}" "${status}"
+end_ts=$(timestamp)
+
+if [[ ${ntotal} -gt 0 ]]; then
+    summary_text="Processed ${ntotal} result tar balls, ${ntbs} successfully"
+    summary_text+=" (${npartialsucc} partial), with ${nquarantined} quarantined"
+    summary_text+=" tar balls, ${ndups} duplicately-named tar balls, and"
+    summary_text+=" ${nerrs} errors."
+    printf -v summary_inner_json \
+        "{\"%s\": \"%s\", \"%s\": %d, \"%s\": %d, \"%s\": %d, \"%s\": %d, \"%s\": %d, \"%s\": %d, \"%s\": \"%s\", \"%s\": \"%s\"}" \
+        "end_ts" "${end_ts}" \
+        "errors" "${nerrs}" \
+        "ndups" "${ndups}" \
+        "npartialsucc" "${npartialsucc}" \
+        "nquarantined" "${nquarantined}" \
+        "ntbs" "${ntbs}" \
+        "ntotal" "${ntotal}" \
+        "prog" "${PROG}" \
+        "start_ts" "${start_ts}"
+    printf -v summary_json "{\"pbench\": {\"report\": {\"summary\": %s}}}" "${summary_inner_json}"
+
+    log_info "${TS}(${PBENCH_ENV}): ${summary_text} -- @cee:${summary_json}"
```

Comment:
So we're writing a log entry to a *file* with `@cee` serialized JSON? How exactly does this become accessible externally? Did I miss something?

Reply:
Only in the test system, see the `alias` line in [`unittests`](https://github.com/distributed-system-analysis/pbench/pull/3299/files#diff-226e4ce92186c9b9ed84c55946bdc4ef8a019d2ca50ba0e4a67f035d026620e3R27).  In normal environments that gets sent out via `syslog` as well.

Reply:
Dang; because I missed that it was going to the `logger` command instead of printed to a file. 😦 PR overload.